ltiauthenticator/lti13/auth.py

```diff
@@ -82,7 +82,27 @@ class LTI13Authenticator(OAuthenticator):
         """,
     )
 
-    def get_handlers(self, app: JupyterHub) -> BaseHandler:
+    tool_name = Unicode(
+        "JupyterHub",
+        config=True,
+        help="""
+        Name of tool provided to the LMS when installed via the config URL.
+
+        This is primarily used for display purposes.
```

Comment:
As a person inspecting this code without using it, this is a very important point answering a question that I've had :)